File: app/src/main/java/com/sysu/edu/todo/TodoActivity.java
package com.sysu.edu.todo;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.sysu.edu.databinding.ActivityTodoBinding;

public class TodoActivity extends AppCompatActivity {


    private InitTodo initTodo;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivityTodoBinding binding = ActivityTodoBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        binding.tool.setNavigationOnClickListener(view -> supportFinishAfterTransition());
        binding.add.setOnClickListener(view -> {
            initTodo.showTodoAddDialog();
        });
        initTodo = new InitTodo(this,binding.fragmentTodo.getFragment());

    }
}

File: app/src/main/java/com/sysu/edu/todo/TodoActivity.kt
package com.sysu.edu.todo

import android.os.Bundle
import com.sysu.edu.BaseActivity
import com.sysu.edu.databinding.ActivityTodoBinding

class TodoActivity : BaseActivity() {
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)
		val binding = ActivityTodoBinding.inflate(layoutInflater).apply {
			toolbar.setNavigationOnClickListener { supportFinishAfterTransition() }
			add.setOnClickListener { (fragment.getFragment<TodoFragment>()).todoManager.showTodoAddDialog() }
		}
		setContentView(binding.root)
	}
}
